buildpack.yml

```
@@ -1,2 +1,2 @@
 mri:
-  version: 2.6.7
+  version: 2.6.8
```

```suggestion
  version: 2.6.9
```
according to the error, `2.6.9` is supported. I guess bumping to it will last longer.